schedule.qmd

```diff
@@ -11,7 +11,7 @@ Here is an example schedule with a morning training and _90_ minutes long, start
 |     |               Topic                 |
 |:---:|:-----------------------------------:|
 |8:50-9:00 | Time to connect. Check audio+video |
-|9:00- 9:05| Sharing rules of participation. CoC. Sharing material |
+|9:00- 9:05| Sharing rules of participation. Code of Conduct. Sharing material |
```

Comment:
this is not a typo but in general I prefer avoiding the abbreviations unless I'm sure every reader will know what that is. :smile_cat: